tests/conftest.py

```diff
@@ -42,6 +43,8 @@
 TESTS_DIR = os.path.dirname(os.path.abspath(__file__))
 DATA_DIR = os.path.join(TESTS_DIR, "data")
 
+FlaskInstrumentor().uninstrument()
+
```

Comment:
This doesn't make us lose much. Mostly just the hooks for tracking what arguments were sent into the flask server, but isn't that important since we can just look at the integration test.